[SplunkPy] Update the README Troubleshooting (#35208)
demisto/content: Packs/SplunkPy/Integrations/SplunkPy/README.md

To resolve this issue, add the IP addresses of Cortex XSOAR8 or Cortex XSIAM to the Splunk Cloud whitelist.
You can find the relevant IP addresses at:
[Cortex XSOAR Administrator Guide](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Administrator-Guide/Enable-Access-to-Cortex-XSOAR) 

> under **Used for communication between Cortex XSOAR and customer resources**. Choose the IP address corresponding to your Cortex XSOAR region.

Reviewer: ```suggestion
Under **Used for communication between Cortex XSOAR and customer resources**. Choose the IP address corresponding to your Cortex XSOAR region.
```

### Fetch Issues

> If you encounter fetch issues and you have enriching enabled, it may have happened as a result of pressing the `Reset the "last run" timestamp` button.  

Reviewer: ```suggestion
If you encounter fetch issues and you have enriching enabled, the issue may be the result of pressing the `Reset the "last run" timestamp` button.  
```
